src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,34 @@
 import React from 'react';
+import { Comment } from '../../types/comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    name,
+    email,
+    body,
+  },
+
+}) => (
+  <div className="CommentInfo">
```

Comment:
```suggestion
  <li className="CommentInfo">
```
that should be `li` (about this https://github.com/mate-academy/react_static-list-of-posts/pull/1583#discussion_r1113222954)